README.md

```diff
@@ -79,3 +79,31 @@ If you encounter any issues or have questions, please file an issue on the [GitH
 
 Keess is open-source software licensed under the MIT license. See the [LICENSE](LICENSE) file for details.
 
+## Local testing
+We will use [kind](https://kind.sigs.k8s.io/) for this
+
+First of all, create 2 clusters:
+```
+make create-local-clusters
```

Comment:
@iblackman  I installed kind using the official guide [here](https://kind.sigs.k8s.io/docs/user/quick-start#installation) using brew.

I'm getting the following error when running `make create-local-clusters`

```bash
make create-local-clusters
Creating cluster "source-cluster" ...
 ✓ Ensuring node image (kindest/node:v1.22.17) 🖼 
 ✗ Preparing nodes 📦  
Deleted nodes: ["source-cluster-control-plane"]
ERROR: failed to create cluster: could not find a log line that matches "Reached target .*Multi-User System.*|detected cgroup v1"
make: *** [create-local-clusters] Error 1
```

Reply:
The problem was that Marcus had the env var `DOCKER_DEFAULT_PLATFORM ` set with value `linux/amd64` probably related to his work to build the **Keess** image to the right platform. But coincidentally **Kind** uses that var to decide which image arch it should download which was causing the **Kind** container to fail to start up so fast that we weren't even able to see the error. 

Reference: https://github.com/kubernetes-sigs/kind/issues/2739